### PR TITLE
test(integration): angular 5.2 + typescript 2.5

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by arturovt** [comment]
@splincode typescript version should be 2.7, see #847

File: integrations/hello-world-ng5/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "karma": "2.0.0",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-coverage-istanbul-reporter": "1.2.1",
+    "karma-jasmine": "1.1.0",
+    "karma-jasmine-html-reporter": "0.2.2",
+    "npx": "^10.2.0",
+    "protractor": "5.1.2",
+    "ts-node": "4.1.0",
+    "tslint": "5.9.1",
+    "typescript": "2.5.3"
```
Comment (arturovt) [inline integrations/hello-world-ng5/package.json]: typescript version should be 2.7, see #847

**Comment by arturovt** [comment]
@splincode I've push angular 6 + typescript 2.7

File: integrations/hello-world-ng5/package.json (hunk 1 above)
Comment (splincode) [reply integrations/hello-world-ng5/package.json]: We need Angular 5 + TypeScript 2.5

**Comment by splincode** [comment]
@arturovt we will add a test for Angular 6 + TypeScript 2.7 later

**Comment by markwhitfeld** [comment]
Let's merge and then work on getting this working!

**Pull request merged**